### PR TITLE
es6 conversion with filter panel in search without jquery

### PR DESCRIPTION
**Pull request by wismer**
as with #1383, some feedback would be great. I'm hoping it keeps the same intended functionality. From what I've been able to test, it appears that way but I am uncertain.

### CODE REVIEW

File: warehouse/static/js/warehouse/utils/search-filter-toggle.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      el.style.display = display;
+    }
+  };
+
+  const toggleEvent = (element, display) => {
+    element.addEventListener("click", togglePanelDisplay.bind(null, display), false);
+  };
+
+  const toggleAccordion = (el) => {
+    el.className = el.className === "accordion" ? "accordion accordion--closed" : "accordion";
```
Comment (berkerpeksag) [inline warehouse/static/js/warehouse/utils/search-filter-toggle.js]: Can this be replaced with `el.classList.toggle("accordion--closed");`?

File: warehouse/static/js/warehouse/utils/search-filter-toggle.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+export default () => {
+  const togglePanelDisplay = (display, event) => {
+    event.preventDefault();
+    var elements = document.querySelectorAll(".-js-dark-overlay, .-js-filter-panel");
+    for (var el of elements) {
+      el.style.display = display;
```
Comment (berkerpeksag) [inline warehouse/static/js/warehouse/utils/search-filter-toggle.js]: I understand that this is basically what the jQuery version does, but perhaps we could defer `el.style.display` part to a new CSS class (something like `.filter-panel--closed { display: none; }`)?

File: warehouse/static/js/warehouse/utils/search-filter-toggle.js (hunk 1 above)
Comment (wismer) [inline warehouse/static/js/warehouse/utils/search-filter-toggle.js]: @berkerpeksag oh good call. much simpler.

File: warehouse/static/js/warehouse/utils/search-filter-toggle.js (hunk 2 above)
Comment (wismer) [inline warehouse/static/js/warehouse/utils/search-filter-toggle.js]: @berkerpeksag That would be easier. Should I run the design/css lead or just put it in anyway?

Comment (berkerpeksag) [inline warehouse/static/js/warehouse/utils/search-filter-toggle.js]: I'd say wait for Nicole or Donald's approval before doing more work on this :)

Comment (wismer) [inline warehouse/static/js/warehouse/utils/search-filter-toggle.js]: oops, I had omitted a few words that really changed the meaning. I meant "run it by the lead"

File: warehouse/static/js/main.js [hunk 3]
```diff
@@ -49,25 +43,13 @@ $(document).ready(function() {
...
-    $(".-js-filter-panel").hide();
-  });
 
   $.timeago.settings.cutoff = 7 * 24 * 60 * 60 * 1000;  // One week
 
-  // document.l10n.ready.then(function() {
-  //   // Format all of the time.relative tags to display relative time.
-  //   $(".-js-relative-time").timeago();
-  // });
+  document.l10n.ready.then(function() {
```
Comment (dstufft) [inline warehouse/static/js/main.js]: This should probably just be deleted, but it certainly shouldn't be uncommented.

File: warehouse/static/js/warehouse/utils/search-filter-toggle.js (hunk 2 above)
Comment (dstufft) [inline warehouse/static/js/warehouse/utils/search-filter-toggle.js]: @nlhkabu would have to answer this question.

**Pull request merged**